tinysemver.py

```diff
@@ -118,6 +118,9 @@ def group_commits(
     patch_commits = []
 
     for commit in commits:
+        # Skip merge commits
```

Comment:
Now that I'm seing this, I might be able to remove that I think, handled by the `--no-merges` above